src/host/server.h

```diff
@@ -166,9 +166,9 @@ class CONSOLE_INFORMATION :
     MidiAudio _midiAudio;
 };
 
-#define CONSOLE_STATUS_WAIT 0xC0030001
-#define CONSOLE_STATUS_READ_COMPLETE 0xC0030002
-#define CONSOLE_STATUS_WAIT_NO_BLOCK 0xC0030003
+#define CONSOLE_STATUS_WAIT ((HRESULT)0xC0030001)

```

Comment:
i vaguely recall that these are NTSTATUSes

Reply:
I did this at some point for some reason to get it to compile, but I forgot why. I reverted it.

I do believe that they're NTSTATUSes as well, but ironically they're mostly used as HRESULTs in most places. I think it would be worth standardizing our code on HRESULT, given that conhost runs in user-mode... right?

Reply:
Seeing the CI failure, I remembered why I did this. 😄
The test fails to build, because an untyped hex number is a signed literal but `HRESULT`s are unsigned.